core/geometry/src/geometry3d/Matrix3d.ts

```diff
@@ -2182,70 +2207,45 @@ export class Matrix3d implements BeJSONFunctions {
     this.createInverseCoffsWithZeros();
     const coffs = this.coffs;
     const inverseCoffs = this.inverseCoffs!;
-    Matrix3d.indexedRowCrossProduct(coffs, 3, 6, inverseCoffs, 0);
-    Matrix3d.indexedRowCrossProduct(coffs, 6, 0, inverseCoffs, 1);
-    Matrix3d.indexedRowCrossProduct(coffs, 0, 3, inverseCoffs, 2);
+    /**
+     * We calculate the inverse using cross products.
+     * Math details can be found at
+     * https://www.chilimath.com/lessons/advanced-algebra/determinant-3x3-matrix/
+     * In summary, if M = [A   B   C] then inverse of M = (1/det)[BxC   CxA   AxB] where det is the
```

Comment:
This line should be changed to

     *                                [   A   ]
     * In summary, if M = [   B   ] then inverse of M = (1/det)[BxC   CxA   AxB] where det is the
     *                                 [   C   ]